docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+# Enabling the OpenStack CLI tool
+
+The OpenStack CLI tool, also known as OpenStack Command Line Client, 
+conveniently provides access to various OpenStack APIs. Using the 
+OpenStack CLI tool, you can remotely create and manage the lifecycle of 
+objects relating, for example, to Compute, Networking, and Storage.
+
+Before installing `openstack` to your local laptop or workstation, you 
+first need to have an OpenStack user in your {{extra.brand}} account. 
+Next, you create and download a special RC file onto your computer, 
+modify it to reflect your OpenStack user's credentials, and source it. 
+Only then will you be able to use `openstack` --- provided it is 
+installed, of course.
```

Comment:
This last sentence sounds a bit weird. I think it's also mildly redundant, so let's just drop it.

Reply:
Well, syntactically it is correct. And although it can be considered as being somewhat redundant, it nevertheless stresses the fact that you *have* to source the RC file *before* you actually use the client. So, in my opinion, the presence and the specific syntax of this particular sentence serve a purpose.

Reply:
It's still oddly backwards. I'd suggest a condensed version: "Only then will you be able to use any installed `openstack` client."

Reply:
Sure, let us go with the condensed version.